massgen/mcp_tools/checklist_tools_server.py

```
@@ -1376,10 +1376,43 @@ def _register_checklist_tool(mcp: fastmcp.FastMCP, specs_path: Path, injection_d
     async def submit_checklist(
         scores: dict,
         report_path: str = "",
+        proposed_criteria: list = None,
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)